includes/reports/overview/list-table/class-sensei-reports-overview-list-table-lessons.php

```diff
@@ -181,6 +182,35 @@ protected function get_row_data( $item ) {
 
 		return $escaped_column_data;
 	}
+
+	/**
+	 * Get the module data for a row.
+	 *
+	 * @param int $lesson_id The lesson post ID.
+	 *
+	 * @return string
+	 */
+	private function get_row_module( int $lesson_id ): string {
```

Comment:
The module data was added via `analysis_overview_column_data`. It made sense to move the logic in this class and deprecate that method.